src/integer_bounds.jl

```diff
@@ -79,7 +79,7 @@ end
     end
     return -1
 end =#
-
+#=
```

Comment:
we can just remove the whole block instead of commenting it

Reply:
Yes, we could but I wanted to leave it until I'm sure that I did not break the previous logic.